contracts/Strategy.sol

```diff
@@ -948,39 +942,38 @@ contract Strategy is BaseStrategy {
             );
     }
 
-    function _setIsWantIncentivised(bool _isWantIncentivised) internal {
-        isWantIncentivised = _isWantIncentivised;
-    }
+    // ----------------- TOKEN CONVERSIONS -----------------
+    function getTokenOutPath(address _token_in, address _token_out)
```

Comment:
I changed all path construction logic in the swap methods for a pure function which does the same.

Reply:
well done